### PR TITLE
[Issue-129] Changed 'Type' to 'Content Type' and changed content types

### LINKED ISSUE #129: Change "Type" to "Content Type" & change content types
** Include...

1. Incident Report
2. Empowering Stories
3. Acts of Allyship

(Will include 4. General News, 5. Resources later)

### PR DESCRIPTION
**Pull request by Nyette**
Resolves #129

### CODE REVIEW

File: src/Components/SideBar/SideBar.jsx [hunk 1]
```diff
@@ -72,9 +72,9 @@ const SideBar = () => {
       <div className={SideBarStyles.ArticleSection}>
         <div className={SideBarStyles.FilterSection}>
           <ButtonMenu
-            buttonLabel="Type"
+            buttonLabel="Content Type"
             optionsTitle="Filter by Content Type"
-            options={["Race-related incident", "Opinion", "Good deed"]}
+            options={["Incident Report", "Empowering Stories", "Acts of Allyship"]}
```
Comment (banhpete) [inline src/Components/SideBar/SideBar.jsx:77]: Does changing the values here for options actually allow it to filter? 

Just looking at the [Airtable](https://airtable.com/tblhquAoHSaDKDnKh/viwlfyCtOKawKNrUh?blocks=bipaNlSeLZIeB6tl3), the types we have are "General News", "Empowering Stories", "Acts of Allyship" and "Incident".

Comment (banhpete) [reply src/Components/SideBar/SideBar.jsx:77]: Also, @SDC-LZC do we also show "Resources", or "Opinions"?

**Review by banhpete — CHANGES_REQUESTED** [review]

File: src/Components/SideBar/SideBar.jsx (hunk 1 above)
Comment (Nyette) [reply src/Components/SideBar/SideBar.jsx:77]: > Does changing the values here for options actually allow it to filter?
> 
> Just looking at the [Airtable](https://airtable.com/tblhquAoHSaDKDnKh/viwlfyCtOKawKNrUh?blocks=bipaNlSeLZIeB6tl3), the types we have are "General News", "Empowering Stories", "Acts of Allyship" and "Incident".

My bad. The filter doesn't work. You're right that the Airtable types don't match the front-end.

Comment (SDC-LZC) [reply src/Components/SideBar/SideBar.jsx:77]: Hey guys -- I think we should show the following (in this order):

- Incident
- Act of Allyship
- Stories of Empowerment
- Resource
- General News

Sorry it was a poorly written ticket.  I can change the Airtable categories to be aligned to the categories written above.

Thank you guys!

**Review by banhpete — APPROVED** [review]
Looks good to me!

**Pull request merged**